wallet/src/test_utils.rs

```diff
@@ -283,7 +283,7 @@ pub fn receive_output_to_address(
         input: vec![],
         output: vec![TxOut {
             script_pubkey: addr.script_pubkey(),
-            value: Amount::from_sat(value),
+            value: value,
```

Comment:
It's the only clippy issue that raises when running locally, maybe this is not dependent on the other PR. 🤔 
```suggestion
            value,
```

Reply:
Weird, I don't get this warning locally, only comment-related ones.